docs/cmd/tkn_resource_list.md

```diff
@@ -0,0 +1,38 @@
+## tkn resource list
+
+Lists `pipelineresources` in a namespace
+
+### Synopsis
+
+Lists `pipelineresources` in a namespace
+
+```
+tkn resource list [flags]
+```
+
+### Aliases
+
+```
+list, ls
+```
+
+### Options
+
+```
+--allow-missing-template-keys        If true, ignores any errors in template when a field or map key is missing. Only applies to golang and jsonpath output formats. (default true)
+ -h, --help                          help for list
+ -o, --output string                 Output format. One of: json|yaml|name|template|go-template|go-template-file|templatefile|jsonpath|jsonpath-file.
+     --template string               Template string or path to template file to use when -o=go-template, -o=go-template-file. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].
+```
+
+### Options inherited from parent commands
+
+```
+--azure-container-registry-config string         Path to the file containing Azure container registry configuration information.
```

Comment:
nit: we need to prepare a follow-up to remove this flag everywhere in the docs :joy_cat:

Reply:
:+1: